src/components/SupportContainer.tsx

```diff
@@ -0,0 +1,100 @@
+// /src/components/SupportContainer.tsx
+
+// React modules
+import React, { useEffect, useState } from "react";
+// array of expert data
+import expertsData from "../../static/data/community-experts.json";
+
+// Component to render community experts and their metadata
+const SupportContainer = () => {
+  // State management
+  const [experts, setExperts] = useState([]);
+
+  // useEffect hook to perform shuffling and data fetching after the component renders
+  // state update will trigger a re-render
+  // empty dependency array to ensure that this effect only runs once
+  useEffect(() => {
+    // Shuffle the array of IExperts randomly
+    const shuffledExperts = shuffleArray(expertsData as IExpert[]);
+    setExperts(shuffledExperts);
+  }, []);
+
+  // return JSX for the SupportContainer's UI
+  return (
+    <div className="support-container">
+      {experts.map((expert) => (
+        <div key={expert.name}>
+          <img
+            className="community-expert-image"
+            src={expert.imageSrc}
+            alt={expert.name}
+          />
+          <br />
+          <span className="name">{expert.name}</span>
+          <br />
+          {expert.channelName && (
+            <>
+              <a
+                href={expert.channelExpertSrc}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {expert.channelName}
+              </a>
+            </>
+          )}
+          <br />
+          {expert.contributionAreas.map(
+            (area: IContributionArea, i: number) => (
+              <span className="contribution-area" key={area.name}>
+                {area.abbreviation +
+                  (i < expert.contributionAreas.length - 1 ? " | " : "")}
+              </span>
+            )
+          )}
+        </div>
+      ))}
+    </div>
+  );
+};
+
+export default SupportContainer;
+
+// Community expert interface to be used with expertsData array
+interface IExpert {
+  name: string;
+  imageSrc: string;
+  channelName: string;
+  channelExpertSrc: string;
+  contributionAreas: IContributionArea[];
+}
+
+// Community expert contribution area interface to be used within experts
+interface IContributionArea {
+  name: "Deneb" | "Vega" | "Vega-Lite";
+  abbreviation: "D" | "V" | "VL";
+}
+
```

Comment:
Propose that we have a lookup array here that normalizes the contribution are data, rather than having in the JSON file, e.g.:

```typescript
const CONTRIBUTION_AREA_MAP: Record<
  ContributionAreaAbbreviation,
  ContributionAreaName
> = {
  D: "Deneb",
  V: "Vega",
  VL: "Vega-Lite",
};
```

Reply:
Ah ok, seeing CONTRIBUTION_AREA_MAP defined here makes things reviewed above clearer.

Understand and agree